platform/features/clock/src/controllers/RefreshingController.js

```diff
@@ -0,0 +1,29 @@
+/*global define*/
+
+define(
+    [],
+    function () {
+        "use strict";
+
+        /**
+         * Continually refreshes the represented domain object.
+         *
+         * This is a short-term workaround to assure Timer views stay
```

Comment:
Is this workaround something that we need to create a separate issue to address?